src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,156 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {

```

Comment:
All in all this is method names which Mykyta recomended me to use
register_addAdultUser_ok
register_addAgeGreaterThanMinAge_ok
register_loginExists_NotOk
register_addNullLogin_NotOk
register_addNullUser_NotOk
register_addNullAge_NotOk
register_addInvalidUsersAge_NotOk
register_passwordLengthIsLessThanMinLength_Ok
register_addNullPassword_NotOk
register_passwordLengthIs6_Ok
register_passwordLengthIsGreaterThanMinLength_Ok